pkg/models/macros.go

```diff
@@ -79,7 +78,7 @@ func InterPolateMacros(queryString string, timeRange backend.TimeRange, pluginCo
 	for key, macro := range macros {
 		matches, err := getMatches(key, queryString)
 		if err != nil {
-			return queryString, errorsource.PluginError(err, false)
```

Comment:
I changed this to return the error instead because it was returning an error with downstream. Not sure if it is the best way though.